cluster/node-pools/master-default/userdata.yaml

```diff
@@ -43,10 +43,9 @@ write_files:
 {{- end }}
       featureGates:
         BoundServiceAccountTokenVolume: {{ .Cluster.ConfigItems.rotate_service_account_tokens }}
-        CSINodeInfo: true
```

Comment:
why is this not needed anymore?

Reply:
It cannot be set to false anymore. It's GA in 1.17.